kategory-docs/docs/docs/README.md

```diff
@@ -4,4 +4,107 @@ title: Quick Start
 permalink: /docs/
 ---
 
-## TODO
\ No newline at end of file
+[![Download](https://api.bintray.com/packages/kategory/maven/kategory/images/download.svg)](https://bintray.com/kategory/maven/kategory/_latestVersion)
+[![Build Status](https://travis-ci.org/kategory/kategory.svg?branch=master)](https://travis-ci.org/kategory/kategory/)
+[![Kotlin version badge](https://img.shields.io/badge/kotlin-1.1.51-blue.svg)](http://kotlinlang.org/)
+[![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+KΛTEGORY is a library for Typed Functional Programming in Kotlin.
+It includes the most popular data types, type classes and abstractions such as `Functor`, `Applicative`, `Monad` and friends empowering users to define pure FP apps and libraries built atop higher order abstractions. Use the below list to learn more about KΛTEGORY's main features.
+
+- [Documentation](http://kategory.io)
+- [Patterns](/docs/patterns/error_handling/)
+- [Type classes](http://kategory.io/docs/typeclasses/functor/)
+- [Data types](http://kategory.io/docs/datatypes/option/)
+- [Effects](http://kategory.io/docs/effects/io/)
+- [Optics](http://kategory.io/docs/optics/iso/)
+
+# Add it to your project
+
+Add it in your root `build.gradle` at the end of repositories.
+
+```groovy
+allprojects {
+    repositories {
+        jcenter()
+        maven { url 'https://kotlin.bintray.com/kotlinx' }
+        maven { url 'https://dl.bintray.com/kategory/maven/' }
+    }
+}
+```
+
+Add the dependencies into the project's `build.gradle`
+
+```groovy
+apply plugin: 'kotlin-kapt' //optional
+apply from: rootProject.file('gradle/generated-kotlin-sources.gradle') //optional
+
+dependencies {
+    compile 'io.kategory:kategory:0.3.10'
+    kapt    'io.kategory:kategory-annotations-processor:0.3.10' //optional
```

Comment:
Isn't `io.kategory:kategory-annotations` missing?

Reply:
`kategory` depends on it so should be transitively included unless I'm missing something from how Gradle works.